sdk/storage/azure-storage-blob/swagger/Update-Codegeneration.ps1

```diff
@@ -0,0 +1,3 @@
+$invokeScript = (Resolve-Path (Join-Path $PSScriptRoot ".." ".." ".." ".." eng scripts Invoke-Codegeneration.ps1)).Path
```

Comment:
While what you have works you can also just do
```
&(Join-Path $PSScriptRoot ".." ".." ".." ".." eng scripts Invoke-Codegeneration.ps1) $PSScriptRoot
```

Reply:
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.3#call-operator-

Reply:
Thanks for this, didn't know about that operator. I'll update these scripts to use this instead as it's much cleaner looking.